### PR TITLE
 ILI-28: Added an ORM_O01 message as an attachment to DocumentSetRequestType

### PR DESCRIPTION
**Pull request by wojtekbu**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java [hunk 1]
```diff
@@ -2,89 +2,70 @@
...
-			DocumentInfo docInfo = buildDocInfo(encounter, patient, docModel);
-			
-			if (!messageUtil.getPatientIdentifier(docInfo).getIdentifierType().getName().equals("ECID")) {
+			DocumentModel clinicalDocModel = clinicalDocBuilder.buildDocument(patient, encounter);
+			DocumentInfo clinicalDocInfo = new DocumentInfo(encounter, patient, clinicalDocModel,
+					"text/xsl", config.getProviderRoot());
+			DocumentData clinicalDoc = new DocumentData(clinicalDocInfo, clinicalDocModel.getData());
+
+			DocumentData msgDoc = null;
+			DocumentModel msgDocModel = ormDocBuilder.buildDocument(encounter);
```
Comment (pgesek) [inline api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java]: This should only happen for orders that are being sent to SCC, see the logic in lab module (it's a concept check)

Comment (wojtekbu) [reply api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java]: I check this on api/src/main/java/org/openmrs/module/xdssender/api/cda/ORM_O01DocumentBuilder.java line 42.
Should this logic be moved or can it stay that way?

File: api/src/main/java/org/openmrs/module/xdssender/api/cda/ORM_O01DocumentBuilder.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package org.openmrs.module.xdssender.api.cda;
```
Comment (pgesek) [inline api/src/main/java/org/openmrs/module/xdssender/api/cda/ORM_O01DocumentBuilder.java]: This has nothing to do with CDA, move to a different package

File: api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java (hunk 1 above)
Comment (pgesek) [reply api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java]: ok that's fine - can you rename msgDoc and msgDocModel to labOrderDoc and labOrderDocModel?

File: api/src/main/java/org/openmrs/module/xdssender/api/cda/ORM_O01DocumentBuilder.java (hunk 2 above)
Comment (wojtekbu) [reply api/src/main/java/org/openmrs/module/xdssender/api/cda/ORM_O01DocumentBuilder.java]: Ok, done.

File: api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java (hunk 1 above)
Comment (wojtekbu) [reply api/src/main/java/org/openmrs/module/xdssender/api/service/impl/XdsExportServiceImpl.java]: Ok, done.

**Pull request merged**